doc/en/components/grids/_shared/cell-selection.md

```diff
@@ -88,10 +88,41 @@ If you want to disable cell selection you can just set `CellSelection` to **none
 
 Below are the methods that you can use in order to select ranges, clear selection or get selected cells data.
 
-<!-- Angular -->
+
+<!-- Angular, WebComponents -->
+
+### Select range
+
+`selectRange` - Select a range of cells with the API. rowStart and rowEnd should use row indexes and columnStart and columnEnd could use column index or column data field value.
```

Comment:
This is missing for Blazor, is it not working?

Reply:
Yes -> https://infragistics.visualstudio.com/NetAdvantage/_workitems/edit/23387/